packages/website/content/docs/examples/fluentui-cra.md

```diff
@@ -0,0 +1,127 @@
+---
+title: 'FluentUI CRA App'
+tags: examples
+category: doc
+---
+
+This example creates a basic React app from scratch using [Create React App] tool, adds an artificial bottleneck
+to it, and then shows you how to configure the build and run flamegrill to see the artificial bottleneck. The full
+code for the app we create and examine in this guide can be found in Flamegrill's GitHub repo in [examples/fluentui-app].
+
+[Create React App]: https://create-react-app.dev/
+[examples/fluentui-app]: https://github.com/microsoft/flamegrill/tree/master/examples/fluentui-app
+
+## Basic app setup
+
+1. Scaffold your app
+
+Create a basic FluentUI app using:
+
+`npx create-react-app my-app --template @fluentui/cra-template`
+
+2. Create sample hotspot
+
+In the app's `src/App.tsx` file, add an `InefficientComponent`:
+
+```tsx
+const InefficientComponent: React.FunctionComponent = (props) => {
+  // This is an abritrary bottleneck to show in flamegraph results.
+  for (let i = 0; i < 100; i++) {
+    console.log(i);
+  }
+  return <div>{props.children}</div>
+}
+```
+
+3. Add hotspot to app
+
+In the same `src/App.tsx` file, use `InefficientComponent`:
+
+ ```tsx
+ export const App: React.FunctionComponent = () => {
+    return 
+      // ... 
+      <InefficientComponent>
+        <img src={logo} alt="logo" />
+      </InefficientComponent>
+```
+
+4. Modify build configuration
+
+In order to generate the flamegraphs for the app, we need to build the app for production. Normally, we'd simply
+run `yarn build` which is an alias for `react-scripts build`, which in turn, under-the-hood calls `webpack` to
+optimize and minify the app. However, for the flamegraphs to be readable and meaningful, we need to disable the
+minification. While `react-scripts` don't allow us to tweak the `webpack` out of the box, it is possible via
+`react-app-rewired` package. With `react-app-rewired` added as a dev dependency, we can tell it to override certain
+configuration values of `webpack` such as turning minification off for the production build.
```

Comment:
Concerning use of `react-app-rewired`, I'm wondering if there are any other alternatives here, like ejecting?

Reply:
Correct, ejecting is also an option; however, my thinking here was to demonstrate that you don't need to go as far as ejecting your project to make it work with flamegrill.